src/components/AboutPage/AboutPage.tsx

```diff
@@ -1,54 +1,197 @@
-import { TextContent, TextList, TextListItem } from '@patternfly/react-core';
 import { useEffect } from 'react';
 
 import { useServiceContainer } from 'hooks/useServiceContainer';
 
+import { Attributes } from 'components/Attributes/Attributes';
+import { AttributesItem } from 'components/Attributes/AttributesItem';
 import { ContentBox } from 'components/ContentBox/ContentBox';
 import { EmptyStateSymbol } from 'components/EmptyStateSymbol/EmptyStateSymbol';
 import { PageLayout } from 'components/PageLayout/PageLayout';
 import { ServiceContainerLoading } from 'components/ServiceContainers/ServiceContainerLoading';
 
-import * as genericSettingsApi from 'services/genericSettingsApi';
+import * as versionApi from 'services/versionApi';
 
 export const AboutPage = () => {
   const pncRepositoryUrl = 'https://github.com/project-ncl/pnc';
   const pncWebUiRepositoryUrl = 'https://github.com/project-ncl/pnc-web-ui-react';
+  const kafkaRepositoryUrl = 'https://github.com/project-ncl/kafka-store';
+  const uiLoggerRepositoryUrl = 'https://github.com/project-ncl/uilogger';
+  const repourRepositoryUrl = 'https://github.com/project-ncl/repour';
+  const bifrostRepositoryUrl = 'https://github.com/project-ncl/bifrost/';
+  const dependencyAnalyzerRepositoryUrl = 'https://github.com/project-ncl/dependency-analysis';
```

Comment:
Consider defining repository URLs as constants outside the component or in a configuration file because we have a lot of them now.